src/SussolReactTable.jsx

```diff
@@ -2,8 +2,25 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Cell, ColumnHeaderCell, EditableCell, Column, Table } from '@blueprintjs/table';
 import FlatButton from 'material-ui/FlatButton';
-import FontIcon from 'material-ui/FontIcon';
 
+/**
+* compare
+*
+* Provides a basic implementation of either a sort ASC or sort DESC comparator.
+* Sort example: 10 will come after 2, not "1, 10, 2"
+*
+* @param {any}     data which can be converted to a string
+* @param {any}     data which can be converted to a string
+* @param {bool}    isAscending: true/false
+* @return {int}    e.g. -1, 0, 1
+*/
+const compare = (a, b, isAscending) => (
+  isAscending
+  ? a.toString().localeCompare(b, undefined, { numeric: true, sensitivity: 'base' })
```

Comment:
localeCompare built into Node? I gotta take note of this one

Reply:
Those mental JS spec people!